pages/news_page.py

```
@@ -48,5 +49,6 @@ def get_first_comment_data(self):
     def get_news_title(self):
         return self.driver.find_element(By.XPATH, news_title).text
 
-
-
+    @allure.step("Open created news by link")
+    def open_page_by_link(self, link):
```

Ruff (0.12.2)

Redefinition of unused `open_page_by_link` from line 12

(F811)